src/test/unittest/ut.c

```diff
@@ -466,3 +467,22 @@ ut_err(const char *file, int line, const char *func,
 
 	errno = saveerrno;
 }
+
+/*
+ * ut_anon_mmap -- mmap of anonymous memory
+ */
+void *
+ut_anon_mmap(const char *file, int line, const char *func, int len)
+{
+        void *base;
+
+        void *dest = util_map_hint(len);
```

Comment:
Looking at your tests, I see no reason for using util_map_hint() here.  Why not just let mmap() pick the address?

Also, I'm not wild about introducing a dependency on a library routine inside the unit test library, so following my suggested makes that a non-issue.